.github/workflows/test-e2e.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
           pip install papermill==2.6.0 jupyter==1.1.1 ipykernel==6.29.5
 
           echo "Install Kubeflow SDK"
-          pip install ./sdk
+          pip install git+https://github.com/kubeflow/sdk.git@main#subdirectory=python
```

Comment:
Just noted, @andreyvelich we need to merge SDK adding PR to the main branch in SDK repository before we cut new 2.0 RC version in this repository.

Reply:
Ah, I missed that.
Thx